### PR TITLE
Add 1.8 federation build/test jobs

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign @madhusudancs

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -4538,7 +4538,7 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
-- interval: 30m
+- interval: 6h
```
Comment (madhusudancs) [inline prow/config.yaml]: Is this change being made for all 1.6 jobs?

Comment (krzyzacy) [reply prow/config.yaml]: yeah, as it's getting older we'd make them run less often

**Comment by krzyzacy** [comment]
thanks! Get it running!

File: prow/config.yaml (hunk 1 above)
Comment (madhusudancs) [reply prow/config.yaml]: Fair enough.

**Pull request merged**